libs/components/angular-tree-component/src/lib/modules/angular-tree/angular-tree-node.component.html

```diff
@@ -18,9 +18,30 @@
   <!-- #region <tree-node-wrapper> custom template --->
   <div
     class="node-wrapper"
+    role="treeitem"
```

Comment:
In [a previous PR](https://github.com/blackbaud/skyux/pull/666), we tried to address accessibility violations by using `aria-owns`. However, that fix only worked for specific modes of the tree view. By moving the `treeitem` role to a different parent element, we are able to address accessibility concerns for all modes of the tree view at once.